sporkfish/evaluator.py

```diff
@@ -234,9 +232,8 @@ def evaluate(self, board: Board) -> float:
         # Assumes:
         # - Chess board implements A1 as first element, H8 as last
         # - Piece square table implements A8 as first element, H1 as last element
-        flip: Callable[[int, int, chess.Color]] = (
-            lambda square, flipped_square, color: square if not color else flipped_square
-        )
+        def flip(square: int, flipped_square: int, color: chess.Color) -> int:
+            return square if not color else flipped_square
```

Comment:
Curious: why decide to use a normal function in the end? (done by ruff automatically?)

Reply:
Ruff agrees with you - doesn't like lambdas. though type hints manually added